connectors/src/connectors/slack/chat/blocks.ts

```diff
@@ -223,3 +223,51 @@ export function makeErrorBlock(
     text: errorMessage,
   };
 }
+
+/**
+ * Creates Slack blocks with buttons for validating a tool execution.
+ * This is used when an agent sends a tool_approve_execution event to Slack.
+ */
+export function makeToolValidationBlock(
+  agentName: string,
+  toolName: string,
+  id: string
```

Comment:
`nit` but to avoid confusion here:
```suggestion
  {agentName, id, toolName}: {agentName: string, id: string, toolName: string}
```